From 241b40331d1b3c4d3e30af2d1dd4fe718fdea5f3 Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Fri, 10 Aug 2012 11:02:04 -0400
Subject: [PATCH] shared/utf8: mark char* as const

Avoids compiler warning:

  src/shared/utf8.c: In function 'ascii_filter':
  src/shared/utf8.c:278:16: warning: assignment discards 'const' qualifier
      from pointer target type [enabled by default]
(cherry picked from commit fd5b4ca11ea2b8a82343bc0db6a2f422beb305f1)
---
 src/shared/utf8.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/shared/utf8.c b/src/shared/utf8.c
index 562c570..0a6a17f 100644
--- a/src/shared/utf8.c
+++ b/src/shared/utf8.c
@@ -194,7 +194,8 @@ char *ascii_is_valid(const char *str) {
 }
 
 char *ascii_filter(const char *str) {
-        char *r, *s, *d;
+        const char *s;
+        char *r, *d;
         size_t l;
 
         assert(str);
